src/main/target/common_post.h

```diff
@@ -278,6 +278,15 @@
 #define USE_SPI_GYRO
 #endif
 
+// Older ACC/GYRO sensors use MPU6500 driver
+#if defined(USE_ACC_SPI_ICM20602) && !defined(USE_ACC_SPI_MPU6500)
+#define USE_ACC_SPI_MPU6500
+#endif
+
+#if defined(USE_GYRO_SPI_ICM20602) && !defined(USE_GYRO_SPI_MPU6500)
+#define USE_GYRO_SPI_MPU6500
+#endif
+
```

Comment:
```suggestion
// Older ACC/GYRO sensors use MPU6500 driver
#if !defined(USE_ACC_MPU6500) && (defined(USE_ACC_ICM20601) || defined(USE_ACC_ICM20602) || defined(USE_ACC_ICM20608G))
#define USE_ACC_MPU6500
#endif

#if !defined(USE_ACC_SPI_MPU6500) && (defined(USE_ACC_SPI_ICM20601) || defined(USE_ACC_SPI_ICM20602) || defined(USE_ACC_SPI_ICM20608G))
#define USE_ACC_SPI_MPU6500
#endif

#if !defined(USE_GYRO_MPU6500) && (defined(USE_GYRO_ICM20601) || defined(USE_GYRO_ICM20602) || defined(USE_GYRO_ICM20608G))
#define USE_GYRO_MPU6500
#endif

#if !defined(USE_GYRO_SPI_MPU6500) && (defined(USE_GYRO_SPI_ICM20601) || defined(USE_GYRO_SPI_ICM20602) || defined(USE_GYRO_SPI_ICM20608G))
#define USE_GYRO_SPI_MPU6500
#endif
```

I'd suggest this to cover all the gyros with and without SPI configs, analog to the MPU6500 defines.

You also need to move these lines up to line 271 or the gating of lines 271-279 might not work correctly.

Reply:
Yes original wanted to do that, but decided to only gate the sensors currently used in targets.

Made the change :+1: